modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/DiagramBuilder.js

```diff
@@ -198,6 +199,30 @@ export default function DiagramBuilder() {
 		setReactFlowInstance(reactFlowInstance);
 	};
 
+	const onNodeDrag = (event, node) => {
+		const reactFlowBounds =
+			reactFlowWrapperRef.current.getBoundingClientRect();
+
+		const position = reactFlowInstance.project({
+			x:
+				event.clientX -
+				reactFlowBounds.left -
+				elementRectangle.mouseXInRectangle,
+			y:
+				event.clientY -
+				reactFlowBounds.top -
+				elementRectangle.mouseYInRectangle,
+		});
+
+		const newElements = elements.filter(
```

Comment:
There are no new elements here, but simply elements. Can you rename all `newElements` in this files to `elements`?

Reply:
Hey, as we talked in person, `filteredElements` is a better name here, as `elements` is already used in the file, even though in a different context.

Reply:
Done :>